spotipy/client.py

```diff
@@ -422,42 +426,6 @@ def playlist_upload_cover_image(self,
                          payload=image_b64,
                          content_type="image/jpeg")
 
-    def user_playlist(self, user, playlist_id=None,
```

Comment:
We want to keep the old endpoints because they still work and people might still be using them

Reply:
They can avoid to update to this version or update their code, using old functions isn't a good idea

Reply:
People should still be able to upgrade, be able to use new features (performance, security, etc), and migrate from old functions to new functions when they find the time to do it. For now, these two endpoints have deprecation warnings